index.js

```diff
@@ -33,6 +33,7 @@ module.exports = {
     Spacer: Layout.Spacer,
     Navigation: Layout.Navigation,
     Content: Layout.Content,
+    MDLUpgrade: require('./lib/MDLUpgrade'),
```

Comment:
I thought we could export it for users wishing some custom markup or something like this.
For example I use code like this if I want link look like a button:

```javascript
                  <a
                    className="mdl-button mdl-button--icon"
                    href={`/download/${id}`}
                    title="Download"
                  >
                    <i className="material-icons">file_download</i>
                  </a>
```

Would be handy if in such cases user could wrap that code inside `MDLUpgrade` which will handle remounting.